provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -78,8 +78,7 @@ def _create_pool_descriptions(self) -> List[PoolDesc]:
 
             min_pool_width = data_units_count + parity_units_count \
                 + spare_units_count
-            if (data_devices_count !=
-                    0) and not (data_devices_count >= min_pool_width):
+            if data_devices_count and (data_devices_count < min_pool_width):
```

Comment:
I think it is good to have an explicit check for integers, `if data_devices_count > 0` in this case. It helps to clearly differentiate a boolean from an integer (I think it is a good practice even though a variable name has `count` in it).

Reply:
Merging the patch.